dev-ml/camomile/camomile-0.8.5-r2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit findlib eutils
+
+DESCRIPTION="Camomile is a comprehensive Unicode library for ocaml"
+HOMEPAGE="https://github.com/yoriyuki/Camomile/wiki"
+SRC_URI="https://github.com/yoriyuki/Camomile/releases/download/rel-${PV}/${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="debug +ocamlopt"
+
+RDEPEND="
+	>=dev-lang/ocaml-3.10.2:=[ocamlopt?]
+	dev-ml/camlp4:=
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+
+	if has_version '>=dev-lang/ocaml-4.05_beta'; then
+		eapply "${FILESDIR}/ocaml405.patch" || die
+	fi
+	if has_version '>=dev-lang/ocaml-4.06' ; then
+		eapply "${FILESDIR}/ocaml406.patch" || die
+	fi
+}
+
+src_configure() {
+	econf $(use_enable debug)
+}
+
+src_compile() {
+	emake -j1 byte unidata unimaps charmap_data locale_data
```

Comment:
Definitely need -j1?

Reply:
Unfortunately yes. That was in the current ebuild; repoman warned me about them so I tried without -j1. Build still fails w/o -j1. I did not do the work of checking if there was already an upstream bug about that.

Reply:
No worries. I think it's fair to say this is an emergency fix. I'll do a bit of research.